packages/testing-helpers/elementUpdated.js

```diff
@@ -0,0 +1,33 @@
+import { nextFrame } from './helpers.js';
+
+const isDefinedPromise = action => typeof action === 'object' && Promise.resolve(action) === action;
+
+/**
+ * Awaits for "update complete promises" of elements
+ * - updateComplete [lit-element]
+ * - componentOnReady() [stencil]
+ *
```

Comment:
maybe add `GluonElement#render`

Reply:
is that an update complete Promise? if so we could add it but pls in separate PR